enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -79,4 +84,62 @@ <h3 class="course-confirmation-title">{{ confirmation_text }}</h3>
       </div>
     </div>
   </div>
+  <div id="course-details-modal" class="modal" aria-hidden="true">
+    <div id="course-details-modal-content" class="modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text">
+      <button class="modal-close-button" id="modal-close-button" aria-label="{{ close_modal_button_text }}">{{ close_modal_button_text }}</button>
+      <header class="wrapper">
+        <div class="image">
+          <img src="{{ course_image_uri }}" alt="{{ course_name }}"/>
+        </div>
+        <div class="details">
+          <h1 id="modal-header-text">{{ course_name }}</h1>
+          <p class="short-description">{{ course_short_description }}</p>
+          <div class="organization">
+            <img src="{{ organization_logo }}" alt="{{ organization_name }}" />
+          </div>
+        </div>
+      </header>
+      <div class="body">
+        <div class="details">
+          <ol>
+            <li>
+              <span class="icon fa fa-tag" aria-hidden="true"></span>
+              <span class="title">{{ price_text }}</span>
+              <span class="text">
+                {% if course_modes|length > 0 %}
+                  {% with course_mode=course_modes.0 %}
+                    {% if course_mode.final_price and course_mode.original_price != course_mode.final_price %}
+                      <strike>{{ course_mode.original_price }}</strike>
+                      <br />
+                      <span class="discount">{{ course_mode.final_price }}</span>
+                    {% else %}
+                      {{ course_mode.original_price }}
+                    {% endif %}
+                  {% endwith %}
+                {% endif %}
+              </span>
+            </li>
+            <li>
+              <span class="icon fa fa-certificate" aria-hidden="true"></span>
+              <span class="title">{{ level_text }}</span>
+              <span class="text">{{ course_level_type }}</span>
+            </li>
+            <li>
+              <span class="icon fa fa-tachometer" aria-hidden="true"></span>
+              <span class="title">{{ effort_text }}</span>
+              <span class="text">{{ course_effort }}</span>
+            </li>
+            <li>
+              <span class="icon fa fa-clock-o" aria-hidden="true"></span>
+              <span class="title">{{ duration_text }}</span>
+              <span class="text">{{ duration }}</span>
+            </li>
+          </ol>
+        </div>
+        <div class="overview">
+          {{ course_overview|safe }}
```

Comment:
@replaceafill The course overview looks good, but it would be great if we could tweak the styling for section headers ("About This Course", "Prerequisites", etc. for the demo course) to match the mock-ups even more closely. According to the mockup they should be bold and using a font that's slightly larger than the text below them.